router/worker.go

```diff
@@ -289,7 +249,7 @@ func (w *worker) WorkerProcess() {
 			}
 
 		case <-timeout:
-			timeout = time.After(jobsBatchTimeout)
+			timeout = time.After(w.rt.reloadableConfig.jobsBatchTimeout)
```

Comment:
Why do have this both here and at line 60?

Reply:
further refactoring is required indeed, but will refrain from doing it now as changes are already enough :)